utils/json/src/signature.rs

```diff
@@ -119,46 +119,22 @@ mod tests {
     use quickcheck_macros::quickcheck;
     use serde_json;
 
-    use super::json::{signature_type::SignatureTypeJson, SignatureJson, SignatureJsonRef};
-
-    #[derive(Clone, Debug, PartialEq)]
-    struct SignatureWrapper {
-        signature: Signature,
-    }
-
-    #[derive(Clone, Debug, PartialEq)]
-    struct SignatureTypeWrapper {
-        sigtype: SignatureType,
-    }
-
-    impl quickcheck::Arbitrary for SignatureWrapper {
-        fn arbitrary(g: &mut quickcheck::Gen) -> Self {
-            let sigtype = SignatureTypeWrapper::arbitrary(g);
-            let signature = Signature::new(sigtype.sigtype, Vec::arbitrary(g));
-            SignatureWrapper { signature }
-        }
-    }
-
-    impl quickcheck::Arbitrary for SignatureTypeWrapper {
-        fn arbitrary(g: &mut quickcheck::Gen) -> Self {
-            let sigtype = g
-                .choose(&[SignatureType::Secp256k1, SignatureType::BLS])
-                .unwrap();
-            SignatureTypeWrapper { sigtype: *sigtype }
-        }
-    }
-
     #[quickcheck]
-    fn signature_roundtrip(signature: SignatureWrapper) {
-        let serialized = serde_json::to_string(&SignatureJsonRef(&signature.signature)).unwrap();
-        let parsed: SignatureJson = serde_json::from_str(&serialized).unwrap();
-        assert_eq!(signature.signature, parsed.0);
+    fn signature_roundtrip(signature: Signature) {
+        let serialized: String =
```

Comment:
Is the `: String` type signature necessary?

Reply:
I know we've been using the explicit type signature in other tests, but apparently we don't need it here!